frontend/src/components/ProfileContent.tsx

```diff
@@ -101,13 +173,39 @@ export default function ProfileContent(props: ProfileContentProps) {
                     <div className={classes.introVideo}>Intro Video</div>
                 </Grid>
             </Grid>
-            <Grid container item alignItems="center" className={classes.fieldsContainer} xs={12}>
-                {profileFields.map(({ title, value }: ProfileField) => (
-                    <Grid key={title} item className={classes.field} sm={12} md={6}>
-                        <Typography>{title}: {value}</Typography>
+            { isEditMode ? null :
+                <Grid id={readOnlyContent} container item alignItems="center" className={classes.fieldsContainer} xs={12}>
+                    <Button variant="contained" color="secondary" className={classes.bottomRight} onClick={handleEditBtnClick}>Edit</Button>
+                    <Grid id={readOnlyContent} container item alignItems="center" xs={12}>
```

Comment:
Two elements with the same id seems like it might cause problems

Reply:
yes, this was a copy and paste error